CoreImportExport/src/au/gov/asd/tac/constellation/importexport/geospatial/SpatialReferenceParameterValue.java

```diff
@@ -83,18 +85,16 @@ protected ParameterValue createCopy() {
     public String toString() {
         String stringSRS = "No Value";
         // TODO: there is a known bug while extracting srs value from CRS.
-        // Ref: #83  
-        return stringSRS;
-//        if (spatialReference == null) {
-//            return stringSRS;
-//        } else {
-//            
-//            try {
-//                stringSRS = spatialReference.getSrs();
-//            } catch (FactoryException ex) {
-//                Exceptions.printStackTrace(ex);
-//            }
-//            return stringSRS;
-//        }
+        // Ref: #83
```

Comment:
@cygnus-x-1 This is a very minor comment but I don't think we really need the reference to #83 anymore.

Reply:
Done.